components/GraphicalMapCard.vue

```diff
@@ -177,82 +186,98 @@ function drawOsaka(elementWidth) {
-        (bounds[0][0] + bounds[1][0]) / 2,
-        (bounds[0][1] + bounds[1][1]) / 2
-      ]
+    // データの中心点を計算
+    // refs: https://qiita.com/yuiken/items/1153922ad20be1d26ced
+    const bounds = d3.geoBounds(json)
+    const center = [
+      (bounds[0][0] + bounds[1][0]) / 2,
+      (bounds[0][1] + bounds[1][1]) / 2
+    ]
 
-      // 投影を処理する関数を用意する。データからSVGのPATHに変換するため。
-      const projection = d3
-        .geoMercator()
-        .center(center)
-        .translate([osakaPrefSize.width / 2, osakaPrefSize.height / 2])
-        .scale(osakaPrefSize.scale)
-      // pathジェネレータ関数
-      const path = d3.geoPath().projection(projection)
-      // これがenterしたデータ毎に呼び出されpath要素のd属性にgeoJSONデータから変換した値を入れて市町村境界描画
-      map
-        .selectAll('path')
-        .data(json.features)
-        .enter()
-        .append('path')
-        .attr('d', path)
-        // 陽性者に対応した色で境界内を塗る
-        .style('fill', function(d) {
-          return popData[d.properties.index][2]
-        })
-      // 左側にデータ表示
-      for (let i = 0; i < 43; i++) {
-        map
-          .append('text')
-          .attr({
-            x: 20,
-            y: i * 13 + 20
-          })
-          .style('font-size', 12 + 'px')
-          .text(popData[i][0] + ':' + popData[i][1])
-      }
+    // 投影を処理する関数を用意する。データからSVGのPATHに変換するため。
+    const projection = d3
+      .geoMercator()
+      .center(center)
+      .translate([osakaPrefSize.width / 2, osakaPrefSize.height / 2])
+      .scale(osakaPrefSize.scale)
+    // pathジェネレータ関数
+    const path = d3.geoPath().projection(projection)
+    // これがenterしたデータ毎に呼び出されpath要素のd属性にgeoJSONデータから変換した値を入れて市町村境界描画
 
-      // 市町村名表示
-      const xhr = new XMLHttpRequest()
-      xhr.onload = function() {
-        const tempArray = xhr.responseText.split('\n')
-        const csvArray = []
-        for (let i = 0; i < tempArray.length; i++) {
-          csvArray[i] = tempArray[i].split(',')
-          const data = csvArray[i]
-          const lonlat = [data[1], data[2]]
-          const xy = projection(lonlat)
-          map
-            .append('text')
-            .attr({
-              x: xy[0] - 15,
-              y: xy[1]
-            })
-            .style('font-size', 10 + 'px')
-            .text(data[0])
-        }
-      }
-      xhr.open('get', 'cityname.csv', true)
-      xhr.send(null)
-    })
-    .catch(function(error) {
-      // エラー処理
-      console.log('in drawOsaka() error:', error)
-    })
+    map
+      .selectAll('path')
+      .data(json.features)
+      .enter()
+      .append('path')
+      .attr('d', path)
+      // 陽性者に対応した色で境界内を塗る
+      .style('fill', function(d) {
+        return popData[d.properties.index].color
+      })
+      .on('mouseover, mousemove', function(d) {
+        tooltip
+          .style('opacity', 0.9)
+          .html(
+            '<strong>' +
+              popData[d.properties.index].name +
+              '</strong><br>' +
+              popData[d.properties.index].count +
+              ' 人'
```

Comment:
```suggestion
              vm.$t('人')
```
こちらも上記の230行目と同じ様に変更が必要となります。